client/src/app/shared/components/chip/chip.component.html

```diff
@@ -0,0 +1,3 @@
+<mat-basic-chip class="os-chip" [disableRipple]="true">
```

Comment:
ah you created this one newly. Since it is just used for tags, you can add more flexible options as stated above.

Maybe do a very variable input: It can be a string, A ViewOrganizationTag or a list of them or so.